.github/workflows/run_tests.yml

```diff
@@ -56,3 +56,4 @@ jobs:
           ./tests/TESTS/build.sh -s DeviceInformation -t GCC_ARM -m NRF52840_DK
           ./tests/TESTS/build.sh -s LinkLoss -t GCC_ARM -m NRF52840_DK
           ./tests/TESTS/build.sh -s LinkLoss -t GCC_ARM -m DISCO_L496AG
+          ./tests/TESTS/build.sh -s FOTA -t GCC_ARM -m NRF52840_DK
```

Comment:
It should also work on disco. There are MCUboot problems but the [mock](https://github.com/noonfom/mbed-os-example-ble/tree/mcuboot/BLE_GattServer_FOTAService/Mock) example works fine locally, i.e. I can transfer a binary into the external flash of my [B-L4S5I-IOT01A](https://os.mbed.com/platforms/B-L4S5I-IOT01A/) using the FOTA service.

**Note:** The disco build is also missing for the Device Information service: #62